app/liquid_tags/codesandbox_tag.rb

```diff
@@ -1,15 +1,19 @@
 class CodesandboxTag < LiquidTagBase
   PARTIAL = "liquids/codesandbox".freeze
+  # rubocop:disable Layout/LineLength
+  REGISTRY_REGEXP = %r{https?://(?:www|app\.)?(?:codesandbox\.io/embed/)(?<video_id>[a-zA-Z0-9-]{0,60})(?:\?)?(?<options>\S+)}
```

Comment:
Note: I did not include a complicated parsing of url parameters in the Main URL regex, opting instead to just capture the whole query with `\S+`. This is because we do validate each parameter passed in within the `parse_options` method.